resources/views/themes/default1/order/show.blade.php

```diff
@@ -111,8 +111,13 @@
                               <input type="hidden" name="domainRes" id="domainRes" value={{$allowDomainStatus}}>
                                 <tbody><tr><td><b>License Code:</b></td>
                                   <td id="s_key" data-type="serialkey">{{($order->serial_key)}}</td>
-                                  <td><span class="label label-success pull-right" id="copied" style="display:none;margin-top:-15px;margin-left:-20px;position: absolute;">Copied</span>
-                                    <span data-type="copy" style="font-size: 15px; pointer-events: initial; cursor: pointer; display: block;" id="copyBtn" title="Click to copy to clipboard"><i class="fa fa-clipboard"></i></span>
+                                  <td> @component('mini_views.copied_flash_text',[
```

Comment:
You are writing more code than you were writing without this component. Defeats the whole purpose of having a reusable component

Reply:
Will be fixed in https://github.com/ladybirdweb/agora-invoicing-community/issues/1093